[ShiftStack: OSDOCS-2025] Full availability zone Cinder volumes (#32120)
openshift/openshift-docs: modules/installation-configuration-parameters.adoc

On clusters that use Kuryr, {rh-openstack} Octavia does not support availability zones. Load balancers and, if you are using the Amphora provider driver, {product-title} services that rely on Amphora VMs, are not created according to the value of this property.
|A list of strings. For example, `["zone-1", "zone-2"]`.

> |`controlPlane.platform.openstack.rootVolume.zones`

Reviewer: Shouldn't it be `compute.platform.openstack.rootVolume.zones`?

Author: 🙊 🙇 Thanks! @mandre

> |`controlPlane.platform.openstack.rootVolume.zones`

Author: ```suggestion
|`compute.platform.openstack.rootVolume.zones`
```

> |For compute machines, the availability zone to install root volumes on. If you do not set this value, the installer selects the default availability zone.

Reviewer 2: nit: I prolly would have used "...do not set this parameter,..." (here and below) because that is the table heading.  Up to you.

> |For compute machines, the availability zone to install root volumes on. If you do not set this value, the installer selects the default availability zone.

Author: ```suggestion
|For compute machines, the availability zone to install root volumes on. If you do not set a value for this parameter, the installer selects the default availability zone.
```

> |For control plane machines,  the availability zone to install root volumes on. If you do not set this value, the installer selects the default availability zone.

Reviewer: Ditto, for compute machines.

> |For control plane machines,  the availability zone to install root volumes on. If you do not set this value, the installer selects the default availability zone.

Author: ```suggestion
|For compute machines,  the availability zone to install root volumes on. If you do not set this value, the installer selects the default availability zone.
```
